tests/jballerina-unit-test/src/test/resources/test-src/expressions/binaryoperations/type-test-expr.bal

```diff
@@ -1227,6 +1227,126 @@ function testBuiltInSubTypeTypeTestAgainstFiniteType() {
     assertFalse(b is Ints);
 }
 
+public type SomeRecord record {
+    int intField;
+};
+
+public function testIfElseWithTypeTest() {
+    SomeRecord? c = {intField: 10};
+    SomeRecord? f = ();
+    any x = true;
+
+    if c is () {
+
+    } else if x is int {
+        c = f;
+    } else if (x is boolean) {
+        foo(c);
+    }
+}
+
+public function testIfElseWithTypeTestMultipleVariables() {
```

Comment:
Shouldn't this and the next two functions be called?

Reply:
I was only checking the FE changes. I will add invokes